addon/model/readers/html-reader.ts

```diff
@@ -35,7 +48,8 @@ export default class HtmlReader implements Reader<Node, ModelNode[], void> {
 
   read(from: Node): ModelNode[] {
     from.normalize();
-    const context = new HtmlReaderContext(this.model);
+    const prefixes = calculateRdfaPrefixes(from);
+    const context = new HtmlReaderContext(this.model, prefixes);
```

Comment:
so what this does is parse the prefixes that may be defined above the element we are reading right?
including going outside of the editor root

I think it's fine when we read the rootnode, but im a bit skeptical about the situation where we read a node that's not the rootnode
I think this currently only occurs in the insert-html command

On first thought I think it will be fine, but it's something to watch out for in any case

Reply:
it should be fine indeed, though perhaps we should hava a param to enable it and default to false? I think it's only necessary when reading the rootNode (it doesn't hurt in other cases afaik, but I don't see much use).

Reply:
id rather just leave it and see if we run into issues
ive had a period where I used a lot of default params but it can get confusing quickly